### DISCUSSION TITLE
Is there a way to migrate my cached music to a new phone?

### DISCUSSION THREAD

**Discussion by iincognito15** [fast4x/RiMusic#4952] [Q&A]
I have the music I've listened to in cache to listen to it later offline, but I want to use a new phone and I want to transfer that "list" of songs that is in the Cache to a new phone, is that possible?

**Comment by riteshrajas** [comment]
Settings > Data > Save back up
 share that file to your new phone
Download the app
Settings > Data > Restore from back up

**Comment by fast4x** [comment]
This save songs list without cached files...
File can't to transfer
